docs/en-us/guide/debug.md

```diff
@@ -414,66 +459,73 @@ startDebugServer();
+Host App set debugMode to true and pass in the bundleUrl generated by front-end webpack for remote wireless debugging, we recommend the host to use input box or scan QR code to pass in.
+
+1. **Android**
 
    ```java
-    // 初始化 hippy 引擎
+    // initialize hippy engine
     HippyEngine.EngineInitParams initParams = new HippyEngine.EngineInitParams();
     initParams.debugMode = true;
-    initParams.remoteServerUrl = "";  // 远程调试 Url
+    initParams.remoteServerUrl = ""; // remote debug bundleUrl
    ```
 
-2. **iOS**：实现 bridgeDelegate 关于调试的代理
+2. **iOS**
 
    ```objective-c
-    /**
-      * ask delegate should bridge start a web inspector
-      * 返回是否开启调试能力
-      */
-    - (BOOL)shouldStartInspector:(HippyBridge *)bridge;
-
-    /**
-      * ask delegate URL for web inspector
-      * 返回调试 bundleUrl
-      */
-    - (NSURL *)inspectorSourceURLForBridge:(HippyBridge *)bridge;
+   - (void)viewDidLoad {
+       // Enable debugging
+       NSDictionary *launchOptions = @{@"DebugMode": @(YES)};
+       NSString *bundleStr = ""; // remote debug bundleUrl
+       NSURL *bundleUrl = [NSURL URLWithString:bundleStr];
+       HippyBridge *bridge = [[HippyBridge alloc] initWithDelegate:self
+                                                         bundleURL:bundleUrl
+                                                    moduleProvider:nil
+                                                     launchOptions:launchOptions
+                                                       executorKey:@"Demo"];
+   }
+   
+   - (BOOL)shouldStartInspector:(HippyBridge *)bridge {
+       return bridge.debugMode;
+   }
+   
+   - (NSURL *)inspectorSourceURLForBridge:(HippyBridge *)bridge {
+       return bridge.bundleURL;
+   }
    ```
 
-## 远程调试支持能力列表
+## List of supported capabilities for remote debugging
 
-> 最低支持版本 2.13.1
+> min support version 2.13.1
 
-|  平台   | HMR & Live-Reload | Elements | Log | Sources | Memory |
-|:-------:|:---:|:-------:|:---:|:------:|:------:|
-| Android |  ✅  |    ✅    |  ✅  |   ✅    |   ✅    |
-|   iOS   |  ✅  |    ✅    |  ❌  |   ❌    |   ❌    |
-
-<br />
-<br />
+| Platform | HMR & Live-Reload | Elements | Log | Sources | Memory |
+|:--------:|:---:|:-------:|:---:|:------:|:------:|
+| Android  | ✅ | ✅ | ✅ | ✅ | ✅ |
+|   iOS    | ✅ | ✅ | ❌ | ❌ | ❌ |
 
-# 框架日志输出
+# framework log output
 
-无论是 hippy-react 还是 hippy-vue 都将和终端通讯的信息进行输出，包含了前终端的节点操作、事件收发。这些日志对于业务调试其实很有帮助，可以让开发了解到前端框架是如何将代码转译成终端可以理解的语法，当遇到问题时应先检查框架通信日志，基本可以定位到大部分问题。
+Both hippy-react and hippy-vue will communicate with the native information output, including the former native node operations, events sent/received. These logs are actually very helpful for business debugging, allowing developers to understand how the front-end framework translates code into a syntax that the native can understand. When you encounter problems, please first check the framework communication logs, as they can basically locate most of the problems.
```

Comment:
including the former native node operations -> including the js-native node operations